src/app/customer-search/customer-search.component.ts

```diff
@@ -3,92 +3,80 @@ import { TestsDataService } from '../services/tests-data.service';
-  public invalidID: boolean = false;
-  public resultsTable = [];
-  public currentPage: number = 0;
+    public idSearch: string;
+    public searchBtn: boolean = true; // Search button is disabled while querying DB
+    public invalidID: boolean = false;
+    public resultsTable: Appointment[] = [];
 
-  constructor(
-    private testDataService: TestsDataService, private tsDataService: TsScreenerDataService, private thsDataService: ThsDataService, private tfiDataService: TfiDataService) { }
+    public currentPage: number = 0;
 
-  public ngOnInit() {
-  }
 
-  // Gets JSON from queryDB() puts it into an array to create the table of appointments
-  // sorts by date, most recent first.
-  public patientSearch() {
-    if (!this.validID()) { this.invalidID = true; return; };
-    this.invalidID = false;
-    this.searchBtn = false;
-    let appts: Object[] = this.getAppointments();
-    this.currentPage = 0;
-    this.resultsTable = appts.map((x) => Object.assign({}, x));
-    this.resultsTable.sort((a, b) => {
-      let date1 = new Date(a.date);
-      let date2 = new Date(b.date);
-      let diff = date2.valueOf() - date1.valueOf();
-      if (diff > 0) {
-        return 1;
-      } else if (diff < 0) {
-        return -1;
-      }
-      return 0;
-    });
-    this.searchBtn = true;
-  }
-  // CHANGE this function to actually load the selected appointment into sessionStorage and tell
-  // audiologist-navigation to change state
-  public loadAppt(appt: Object) {
-    if (appt['id'] === '') { return; }
-    // for each dataService saveData
-    console.log('appt: ' + appt['date']);
-  }
-  // pagination functions
-  public prevPage(amt: number) {
-    while (this.currentPage > 0 && amt > 0) {
-      this.currentPage--;
-      amt--;
+    constructor(
+        private testDataService: TestsDataService, private tsDataService: TsScreenerDataService, private thsDataService: ThsDataService, private tfiDataService: TfiDataService, private customerSearchService: CustomerSearchService) { }
+
+    public ngOnInit() {
+    }
+
+    // Gets JSON from queryDB() puts it into an array to create the table of appointments
+    // sorts by date, most recent first.
+    public patientSearch() {
+        if (!this.validID()) { this.invalidID = true; return; };
+        this.invalidID = false;
+        this.searchBtn = false;
+        this.getAppointments();
+
+        this.searchBtn = true;
     }
-  }
-  public nextPage(amt: number) {
-    while ((this.currentPage + 1) * 10 < this.resultsTable.length && amt > 0) {
-      this.currentPage++;
-      amt--;
+    // CHANGE this function to actually load the selected appointment into sessionStorage and tell
+    // audiologist-navigation to change state
+    public loadAppt(appt: Appointment) {
+        if (appt['id'] === '') { return; }
+        // for each dataService saveData
+        console.log('appt: ID-"' + appt.appointmentid + '"      Audiologist-"' + appt.authorityname + '"');
```

Comment:
Unsure why the client needs to log to console, If this is for logging purposes, perhaps there's a better way?

Reply:
This is a placeholder console call to verify that the Appointment details are properly held. It will be replaced once the Appointment search is connected to the Appointment summary components are connected